crates/core/src/constant.rs

```diff
@@ -0,0 +1,50 @@
+use std::fmt::Display;
```

Comment:
This file has simply been extracted out of `binding.rs`, so that one only contains the trait now.